lib/Configuration.php

```diff
@@ -102,8 +102,14 @@ class Configuration
     public function __construct()
     {
         $config     = array();
-        $configFile = PATH . 'cfg' . DIRECTORY_SEPARATOR . 'conf.php';
-        $configIni  = PATH . 'cfg' . DIRECTORY_SEPARATOR . 'conf.ini';
+        $basePath   = PATH. 'cfg' . DIRECTORY_SEPARATOR;
+        $configIni  = $basePath . 'conf.ini';
+
+        if (getenv('CONFIG_PATH') !== false) {
+            $configFile = getenv('CONFIG_PATH');
+        } else {
+            $configFile = $basePath . 'conf.php';
+        }
```

Comment:
@m1cr0man: Sorry, I did write the above review on Friday, but apparently hadn't submitted the whole review.

Reply:
@elrido The way I'm defining CONFIG_PATH here is that it isn't the base path to a config, it's the absolute path to the file. The logic on line 115 won't work if the file isn't called `conf.ini` exactly, which it isn't in Nix for example.

It would take a few changes to the rename logic to make things work. First off I'd need to check if the configFile ends in the .ini extension (or'd with the current condition), then check if the file is writable, then make the changes. Also the logic for the sample INI should probably move to the outer scope.

Actually it just occurred to me that I can leave that part untouched and just add a special case for handling CONFIG_PATH being ini and not php. I'll commit that for your review.